include/wx/gtk/private/value.h

```diff
@@ -17,6 +17,11 @@
 class wxGtkValue
 {
 public:
+    explicit wxGtkValue()
+        : m_val(G_VALUE_INIT)
```

Comment:
`G_VALUE_INIT` is a braced-init-list, using it like this requires C++11. Same for the ctor below.

Reply:
Thanks for noticing this. AFAICS the only C++98-compatible way to fix this is to use `memset(&m_val, 0, sizeof(m_val));`, do you see anything less clumsy?

Reply:
That's the only way I'm aware of.